app/assets/stylesheets/components/headers.scss

```diff
@@ -1,5 +1,13 @@
 @import '../config/import';
 
+.crayons-page-header {
+  padding: var(--su-3) var(--su-3) var(--su-1);
+
+  @media (min-width: $breakpoint-m) {
+    padding: 0;
+  }
+}
+
```

Comment:
I hope so! I think it can replace the multiple pages where we're currently using utility classes to render the spacing. I think it would be more reliable to use this class than those utility classes.

I won't make all of those updates in this PR but create a card for a PR focused on that.